test/.eslintrc

```diff
@@ -33,10 +33,16 @@
 
     // @easypost/eslint-config-easypost-base assumes all tests are jest. so turn these rules off
     "jest/consistent-test-it": 0,
-    "jest/valid-expect": 0
+    "jest/valid-expect": 0,
+    "global-require": "off"
   },
 
   "globals": {
     "expect": true
+  },
+  "parser": "@babel/eslint-parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
```

Comment:
Issue: Please pin this `ecma` version. Without doing so, it will lead to lint errors when new rules or syntax becomes available.

Reply:
Pin to `13`